### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by zacheliason**
This pull request fixes the two high priority bug fixes you added to Trello!

### CODE REVIEW

File: front_end/content.py [hunk 1]
```diff
@@ -1638,11 +1638,12 @@ def save_submission(self, course, assignment, exercise, user, code, text_output,
...
             if partner_id:
                 submission_id = self.get_next_submission_id(course, assignment, exercise, partner_id)
-                self.execute(sql, [int(course), int(assignment), int(exercise), partner_id, int(submission_id), code, text_output, image_output, passed, datetime.now(), user, tests])
+                self.execute(sql, [int(course), int(assignment), int(exercise), partner_id, int(submission_id), code, text_output, image_output, passed, datetime.now(), user])
 
                 if len(tests) > 0:
                     for test in tests:
-                        self.execute(test_sql, [int(course), int(assignment), int(exercise), user, int(submission_id), test["text_output"], test["image_output"]])
+                        self.execute(test_sql, [int(course), int(assignment), int(exercise), partner_id, int(submission_id), test["text_output"], test["image_output"]])
+
```
Comment (zacheliason) [inline front_end/content.py:1646]: Allows pair programming submissions to be saved

File: front_end/content.py [hunk 2]
```diff
@@ -1925,6 +1926,11 @@ def delete_exercise_submissions(self, exercise_basics):
                           AND assignment_id = ?
                           AND exercise_id = ?''', (course_id, assignment_id, exercise_id, ))
 
+        self.execute('''DELETE FROM submission_outputs
+                        WHERE course_id = ?
+                          AND assignment_id = ?
+                          AND exercise_id = ?''', (course_id, assignment_id, exercise_id, ))
+
     def create_scores_text(self, course_id, assignment_id):
```
Comment (zacheliason) [inline front_end/content.py:1934]: Unrelated bug fix for deleting submissions

File: front_end/css/styles.css [hunk 3]
```diff
@@ -453,8 +453,8 @@ td > button {
 }
 
 .passing-btn-color {
-    background-color: #00d1b2; /* teal */
-    border-color: #00d1b2; /* teal */
+    background-color: #1AB28C; /* teal */
+    border-color: #1AB28C; /* teal */
```
Comment (zacheliason) [inline front_end/css/styles.css:457]: Reconciles the two teal colors

File: front_end/html/edit_exercise.html [hunk 4]
```diff
@@ -795,7 +795,6 @@ <h2>Create exercise</h2>
                 output_button.innerHTML = `Test ${test_index + 1} Output`
                 var output_content = document.createElement("div")
                 output_content.id = `test_${test_index}`
-                output_content.style = 'display:none'
                 var test_p_wrapper = document.createElement('p')
```
Comment (zacheliason) [inline front_end/html/edit_exercise.html:798]: Test output is displayed by default

File: front_end/html/exercise_shared.html [hunk 5]
```diff
@@ -205,7 +205,7 @@ <h6>Copy exercise within the same assignment:</h6>
                             test_message = (tests[{{i}}].passed ? "passed" : "did not pass");
                             {% if exercise_details["tests"][i]["test_instructions"] == "" %}
                                 test_output += `<pre class='${button_bg_color}' style='cursor:pointer;' onclick='showTest(${tests[{{i}}].test})'>Test {{i + 1}} ${test_message}.</pre>`;
-                                test_output += `<div id="test_${tests[{{i}}].test}" style="display:none"><h6>Your output:</h6>\n\n`;
+                                test_output += `<div id="test_${tests[{{i}}].test}" style="display:block"><h6>Your output:</h6>\n\n`;
```
Comment (zacheliason) [inline front_end/html/exercise_shared.html:208]: Test output is displayed by default.

File: front_end/html/exercise.html [hunk 6]
```diff
@@ -282,15 +282,11 @@ <h6 style="display:inline">Using Pair Programming:</h6>
                 </label><br />
 
                 <div id="invalid_partner"></div>
-                <div style="margin: 15px 0;" class="select is-grey space_div" >
-                    <select onchange="showUsers(this.value)" id="pair_programming" name="pair_programming" class="edit-select">
-                        <option value="Yes">Yes</option>
-                        <option value="No" selected>No</option>
-                    </select>
-                </div>
```
Comment (zacheliason) [inline front_end/html/exercise.html:285]: Removes select input for pair programming, asks user to leave blank if not using pair programming instead

File: front_end/html/exercise_shared.html [hunk 7]
```diff
@@ -442,8 +442,8 @@ <h6>Copy exercise within the same assignment:</h6>
                      var modalSubmitButton = document.getElementById("modal_submit_button");
 
                      modalSubmitButton.onclick = function() {
-                       let partner_key = {"partner_key": document.getElementById("partner_key").value};
-                       if (document.getElementById("pair_programming").value == "Yes") {
+                       let partner_object = {"partner_key": document.getElementById("partner_key").value};
+                       if (partner_object.partner_key != "") {
                          $.post("/check_partners/{{ course_basics['id'] }}", partner_key,
                             function(data, status) {
                                 if(!(JSON.parse(data))) {
```
Comment (zacheliason) [inline front_end/html/exercise_shared.html:449]: These changes are also needed to be able to remove the select input.

**Review by srp33 — APPROVED** [review]
